pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -551,12 +551,18 @@ private static int GetWaitCycles(int millis = WAIT_TIME)
                 waitCycles *= Program.UnitTestTimeoutMultiplier;
             }
 
-            // Wait a little longer for debug build. (This may also imply code coverage testing, slower yet)
+            // Wait a little longer for debug build.
             if (ExtensionTestContext.IsDebugMode)
             {
                 waitCycles = waitCycles * 4;
             }
 
+            // Code coverage testing may be slower yet.
+            if (Helpers.RunningResharperAnalysis)
+            {
+                waitCycles *= System.Diagnostics.Debugger.IsAttached ? 2 : ExtensionTestContext.IsDebugMode ? 4 : 16;
```

Comment:
I don't totally understand all of these options. These multipliers are additive and IsDebugMode and Debugger.IsAttached get added above. Let's also make the code above consistently use

waitCycles *= multiplier;

instead of sometimes:

waitCycles = waitCycles * multiplier;

Reply:
Added a comment to clarify:
// Code coverage testing may be slower yet. But don't apply multipliers that have already been applied above.

Reply:
Maybe this could be made clearer by having a "waitMultiplier" variable that accumulates above and just gets set to 16 in this case? And then the final line could be "return waitCycles * waitMultiplier;